azure-devops/projects/io-app-projects/secrets.tf

```
@@ -0,0 +1,19 @@
+module "secrets" {
+  source = "../../modules/secrets/"
+
+  resource_group = "io-p-rg-operations"
+  keyvault_name  = "io-p-kv-azuredevops"
```

```suggestion
```suggestion
  source = "git::https://github.com/pagopa/azurerm.git//key_vault_secrets_query?ref=v1.0.11"

  resource_group = "io-p-rg-operations"
  key_vault_name = "io-p-kv-azuredevops"
```
please use shared module
```